ddp-workspace/projects/ddp-dsm-ui/src/app/services/auth.service.ts

```diff
@@ -210,24 +187,22 @@ export class Auth {
           jsonData.forEach((val) => {
             this.realmList.push(NameValue.parse(val));
           });
+
+          // this.realmList = [{name: 'Singular', value: 'Singular'}];
+          const navigateUri = this.realmList.length > 1 ? 'pickStudy' : this.realmList[0].name;
```

Comment:
it would be nice to keep custom strings as constants

Reply:
Done